eppo/src/main/java/cloud/eppo/android/EppoClient.java

```diff
@@ -36,14 +36,16 @@ public class EppoClient {
     private boolean isGracefulMode;
     private static EppoClient instance;
 
-    // Useful for testing in situations where we want to mock the http client
+    // Useful for testing in situations where we want to mock the http client or configuration store
     private static EppoHttpClient httpClientOverride = null;
+    private static ConfigurationStore configurationStoreOverride = null;
```

Comment:
give the ability for tests to use reflection to inject a configuration store without complicating the public "API" of this class.